docker/HOWTO.md

```diff
@@ -0,0 +1,35 @@
+# WOVN.php debug environment
+
+## Start
+1. build docker image  
+Build docker for the first time.
+```
+docker-compose build
+```
+
+2. Set wovn.ini  
+Rewrite wovn.ini.sample with your config.
+
+3. Run docker  
+Run the following command.  
+You should be able to see the content when you access `localhost`.
+```
+docker-compose up -d
+```
+
+## Stop
+```
+docker-compose rm -fs
+```
+
+## Use local html-swapper
+1. wovn.ini  
+Add the following to wovn.ini.sample
+```
+api_url = "http://host.docker.internal:3001/v0/"
```

Comment:
This means we must add `host.docker.internal` to our `/etc/hosts` file, right?

Reply:
`host.docker.internal` means host PC's `localhost`.
So,  `http://host.docker.internal:3001/v0/` means `localhost:3001/v0/`.
If you set `api_url = "localhost:3001/v0/"`, it means docker's localhost.

Reply:
Ah I see, it is understood by docker as the localhost of the machine running the docker container?

Reply:
Yes

Reply:
Ok, thanks.